app/models/participant_profile/ecf.rb

```diff
@@ -73,6 +71,11 @@ def relevant_induction_record(lead_provider:)
         .first
     end
 
+    def self.start_term_options(cohort)
```

Comment:
This probably belongs on a cohort because it's not consuming anything related to participant profiles.